src/api/script/delayed_job_monitor.rb

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby.ruby2.4
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
```

Comment:
Why do we need to load the environment?

Reply:
Otherwise errbit wouldn't be triggered, right?

Reply:
Ah. Clever.